rotkehlchen/api/v1/fields.py

```diff
@@ -864,3 +864,20 @@ def _deserialize(
             raise ValidationError('List cant be empty')
 
         return result
+
+
+class EnsNameField(fields.String):
+    def _deserialize(
+            self,
+            value: str,
+            attr: Optional[str],  # pylint: disable=unused-argument
+            data: Optional[Mapping[str, Any]],
+            **_kwargs: Any,
+    ) -> str:
+        if not isinstance(value, str):
+            raise ValidationError('Ens name has to be a string')
```

Comment:
Since you're inheriting from `fields.String` I don't think this check is needed.